.travis.yml

```diff
@@ -1,30 +1,32 @@
 dist: trusty
 sudo: required
+services:
+  - docker
 language: node_js
 node_js:
-- '8'
+  - "8"
 os:
-- linux
+  - linux
 env:
   global:
   - DBUS_SESSION_BUS_ADDRESS=/dev/null
   - DISPLAY=:99.0
   - CHROME_BIN=chromium-browser
-  - secure: CZd2EJc1viLeNjdyBA3EJqeAz6RHKhupZfjvG70RhKkoNGVyzCcWTYyZq3wgqaO0TJQ31bdbOwvCmtRX+xgEFyt3RqclZPmQudbooiz2taFzTWez7qRKGHoRmXyPGvtu8vgQXXFI2iNetpumPh6CCud0vcmGidZwzkgg3D3+fBtr/8sWpz7kUXEuE8IslmbtYWzGMsrjjJz7mpiU2KrrL0lkCaDZFFnzZXE16Zyq3YT/Lx8PdO4Dv67T7ht5/UDKhztzOMR6at2FB/J2oaaeCQYYSkzBfdIYABAAcQ13Z9lmSuVDJaDw8knpsMrV7y8WtXfZ71JYWAj5kp9FFChWZqcnWFZ8yFUA+fkFzPJ+8rSjzEgqp8+ouSseaZ8jGymCSBctONdT8ykCpeXHq6ReQ6f7smVWO3n1QtiWr/DrdF2DItbFuClV10PUIGw5FIxwxP1q0Yjmp6zfTG/7s/L0Gf8NidMhabeLIVL1dR1GtMNvHos/OFMMp0kFvYCwfnGmHj5fV7RZI7K9DvZtor6RcqFkqvIsvuH5FMgcqbYsMH8rjoUt7s7IFahQbsvm0B8TIkCLjmR3PbBFGxFVHJjTKP1Dtkf2yOVcb9r8e66UuaGl6+9yn/j3rFHx7syyT2q/isC6PuH5cf+BbEOJ7JbXOEvG2bkP93P9w0vccziK/98=
```

Comment:
Don't remove these - these are the docker credentials.

Reply:
I guess they are `$DOCKER_USER` and `$DOCKER_PASS`.